christina-cook/rancid-tomatillos
# Refactor css

##Is this a fix or a feature?
This is a refactor.

##What is the change/fix?
Updated and refactored CSS styles on all components.

##Where should the reviewer start?
Review the SCSS file for each component

##How should this be tested?
Run the server and make sure everything still looks good.

##Screenshots(if appropriate)
